src/olympia/amo/tests/__init__.py

```diff
@@ -766,6 +794,8 @@ def version_factory(file_kw=None, **kw):
     max_app_version = kw.pop('max_app_version', '5.0.99')
     version = kw.pop('version', '%.1f' % random.uniform(0, 2))
     application = kw.pop('application', amo.FIREFOX.id)
+    if not kw.get('license') and not kw.get('license_id'):
+        kw['license'] = license_factory(**(kw.get('license_kw', {})))
```

Comment:
Well, it shouldn't create more edge cases since we do have the concept of built-in licenses ? But if it's too complex let's drop it